scale-down-nodes-vsphere.yml

```diff
@@ -0,0 +1,103 @@
+---
+- hosts: localhost
+
+  vars:
+    workerContent : "{{ lookup('file', '{{ playbook_dir }}/install-dir/worker.64') }}"
+    folder : "/{{ vcenter.datacenter }}/vm/openshift/{{ config.clusterName }}"
+    datastore: "{{ vcenter.datastore }}"
+    govc_file: "/usr/local/bin/govc"
+
+
+  environment:
+    # KUBECONFIG: ./kubeconfig
+    # KUBECONFIG: /root/.kube/config
+    KUBECONFIG: /root/ocp4-vsphere-upi-automation/install-dir/auth/kubeconfig
+
+  tasks:
+    - name: OCPNODES 1.0 | Cordon Nodes
+      shell: oc adm cordon "{{ item.name }}.{{ dns.clusterid }}.{{ dns.domain }}"
```

Comment:
Shouldn't this also include `oc adm manage-node --schedulable=false <node>`?

Reply:
Cordon is the same thing. Shouldn't be required.